src/core/bridge.ts

```diff
@@ -19,6 +27,23 @@ export interface BridgeCallback {
   (err?: Error, res?: Results): void;
 }
 
+export class BridgeTimeout extends Error {
+  constructor (err: string) {
+    super(err);
+  }
+}
+
+export class BridgeError extends Error {
+  constructor (statusText: string, public status: number, public data: any) {
+    super(statusText);
+    Object.setPrototypeOf(this, BridgeError.prototype);
```

Comment:
We need this one or else the prototype is going to be the same as Error, and the getter I defined lower down won't work.

Reply:
:+1: